vendor/assets/javascripts/react-chosen.js

```diff
@@ -0,0 +1,69 @@
+(function(window, React, $) {
```

Comment:
But then we can potentially have different react version in the dummy app (and the tests) and the actual app where ui_c is used, right?